src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -117,6 +113,15 @@ protected override void OnScriptFileChanged(FileSystemEventArgs e)
             }
         }
 
+        private void Shutdown()
```

Comment:
_shutdown is a general method thought that gets called for reasons, and those reasons may expand in the future. So is it correct to have this specific error message on this path?